index.html

```diff
@@ -920,5 +920,69 @@ <h3>
               </dd>
             </dl>
 </section>
+<section>
+    <h3>
+      {{RTCDataChannel}} extensions
+    </h3>
+    <p>This document extends {{RTCDataChannel}} by making it <a data-cite="!HTML/#transferable-objects">transferable</a>.</p>
+      This allows sending and receiving messages outside the context the connection was created, for instance in workers or third-party iframes.</p>
```

Comment:
I think this paragraph can be extended a little bit to give more context. In particular, something like:

An RTCDataChannel can be transferred immediately after being created. As soon as incoming data may be present on the RTCDataChannel, it is no longer transferable.

It is possible to call send() on a DataChannel before transferring it.

Reply:
For simplicity, calling send() will not make the data channel transferable anymore.

Reply:
Agree that allowing calling `send()` should probably be disallowed for simplicity. I'm more worried about data that may be in the receive pipeline.  Data can be sent by the other peer as soon as the SctpTransport is established and the data channel is open. Why force the queued data to be thrown away?

Reply:
As discussed during the call, we make sure no message data is lost in the transfer.
This is explicitly stated in the latest PR.